tests/conftest.py

```diff
@@ -58,54 +66,26 @@ def db_dsn(
     # Assign a temporary database for sqlite
     if database_dsn == "sqlite":
         database_dsn = f"sqlite:///{tmp_path / 'test.db'}"
-    yield database_dsn
-
-
-@pytest.fixture()
-def db_engine(
-    db_dsn: str, request: pytest.FixtureRequest
-) -> Generator[Engine, None, None]:
-    """Create a database engine and optionnally by default initialize the schema."""
-    engine = create_engine(db_dsn)
-    if request.param:
-        init_database(engine)
-    yield engine
-    if request.param:
-        BaseModel.metadata.drop_all(engine)
```

Comment:
Why delete this ? Where do you clear the database now ?

Reply:
My bad, these are just moved to `tests/unit/conftest.py`

Reply:
Yeah, db engine and session only make sense in the unit tests context in my opinion!